flash/core/utilities/imports.py

```diff
@@ -60,6 +60,11 @@ def _compare_version(package: str, op, version) -> bool:
         return False
     try:
         pkg_version = Version(pkg.__version__)
+    except AttributeError:
+        # In case the module doesn't have __version__ attribute (example: baal)
+        import pkg_resources
+
+        pkg_version = Version(pkg_resources.get_distribution("baal").version)
```

Comment:
```python
>>> import baal
>>> baal.__version__
----> 1 baal.__version__

AttributeError: module 'baal' has no attribute '__version__'
```

(interestingly, this error is only with 1.5.2 version of BaaL) - checked with 1.3.2, it wasn't raised there.